ComponentKit/DataSources/CKComponentBoundsAnimation+UICollectionView.mm

```diff
@@ -33,69 +33,72 @@ @implementation CKComponentBoundsAnimationCollectionViewContext
   NSInteger _numberOfSections;
   std::vector<NSUInteger> _numberOfItemsInSection;
   NSDictionary *_indexPathsToSnapshotViews;
+  NSDictionary *_supplementaryElementIndexPathsToSnapshotViews;
   NSDictionary *_indexPathsToOriginalLayoutAttributes;
+  NSDictionary *_supplementaryElementIndexPathsToOriginalLayoutAttributes;
 }
 
-- (instancetype)initWithCollectionView:(UICollectionView *)cv
+- (instancetype)initWithCollectionView:(UICollectionView *)collectionView
```

Comment:
Thanks for cleaning up the abbreviations! This code is much easier to reason about now.